### PR TITLE
API: Platforms

### PR DESCRIPTION
**Pull request by kelunik**

### CODE REVIEW

File: src/main/java/edu/kit/ipd/crowdcontrol/objectservice/database/operations/PlatformOperations.java [hunk 1]
```diff
@@ -2,22 +2,58 @@
...
+    }
+
+    /**
+     * Returns a single platform.
+     *
+     * @param id ID of the platform
+     *
+     * @return The platform.
+     */
+    public Optional<Platform> get(String id) {
```
Comment (LeanderK) [inline src/main/java/edu/kit/ipd/crowdcontrol/objectservice/database/operations/PlatformOperations.java]: getPlatform

File: src/main/java/edu/kit/ipd/crowdcontrol/objectservice/database/operations/PlatformOperations.java [hunk 2]
```diff
@@ -2,22 +2,58 @@
...
     /**
+     * Returns a range of platforms starting from {@code cursor}.
+     *
+     * @param cursor pagination cursor
+     * @param next   {@code true} for next, {@code false} for previous
+     * @param limit  number of records
+     *
+     * @return List of platforms.
+     */
+    public Range<Platform, String> all(String cursor, boolean next, int limit) {
```
Comment (LeanderK) [inline src/main/java/edu/kit/ipd/crowdcontrol/objectservice/database/operations/PlatformOperations.java]: getPlatformsFrom

File: src/main/java/edu/kit/ipd/crowdcontrol/objectservice/rest/resources/PlatformResource.java [hunk 3]
```diff
@@ -0,0 +1,44 @@
+package edu.kit.ipd.crowdcontrol.objectservice.rest.resources;
```
Comment (marcelhollerbach) [inline src/main/java/edu/kit/ipd/crowdcontrol/objectservice/rest/resources/PlatformResource.java:1]: Same like above with the method names

Comment (marcelhollerbach) [inline src/main/java/edu/kit/ipd/crowdcontrol/objectservice/rest/resources/PlatformResource.java:1]: I mean in this file...

Comment (kelunik) [inline src/main/java/edu/kit/ipd/crowdcontrol/objectservice/rest/resources/PlatformResource.java]: Resources are always named like the HTTP method.

Comment (kelunik) [inline src/main/java/edu/kit/ipd/crowdcontrol/objectservice/rest/resources/PlatformResource.java:1]: It's currently consistent with all other classes in `rest.resources`. If we change it, it should be changed in a separate PR for all classes.

**Comment by kelunik** [comment]
Ready to merge?

**Comment by LeanderK** [comment]
ready to merge!
